tests/admin/test_controller.py

```diff
@@ -99,10 +98,34 @@ def test_edit(self):
                 ("summary", "<p>New summary</p>")
             ])
             response = self.manager.admin_work_controller.edit(lp.data_source.name, lp.identifier.identifier)
+            eq_(200, response.status_code)
             eq_("Young Adult", self.english_1.audience)
             assert 'Young Adult' in self.english_1.simple_opds_entry
             assert 'Adults Only' not in self.english_1.simple_opds_entry
 
+        with self.app.test_request_context("/"):
+            # Change the summary again
+            flask.request.form = ImmutableMultiDict([
+                ("title", "New title"),
+                ("audience", "Young Adult"),
+                ("summary", "abcd")
+            ])
+            response = self.manager.admin_work_controller.edit(lp.data_source.name, lp.identifier.identifier)
+            eq_(200, response.status_code)
+            eq_("abcd", self.english_1.summary_text)
+            assert 'New summary' not in self.english_1.simple_opds_entry
+
```

Comment:
That is, in my previous comment I'm saying that this might work by accident. Unless you do something with the old description I think it might still be hanging around, and at some point in the future it might come back.